dune-project

```diff
@@ -1,4 +1,38 @@
-(lang dune 1.8)
+(lang dune 1.10)
 (name dune)
 
-(implicit_transitive_deps false)
\ No newline at end of file
+(implicit_transitive_deps false)
+
+(license MIT)
+(authors "Jane Street Group, LLC <opensource@janestreet.com>")
```

Comment:
The authors list is an interesting one. I think we should encourage the listing of individual contributions as well in order to keep this accurate. What do you think about having a general `author` schema that could be

```
(maintainers @diml @rgrinberg)
(authors "Jane Street Group <opensource@janestreet>" (gitlab @avsm))
```

The idea here is that maintainers are always authors, and that we are flexible about how to identify someone. A string starting with @ is assumed to be github by default.

(this can probably be deferred for a future PR, but I just thought about it now!)

Reply:
I always wonder what to put in this field BTW. For instance, should it be every single person who submitted a PR? The "Insights" tab is much more precise in this regard.

Reply:
It's a value judgement of "significant contributions".  So a typo fix doesnt cut it, but a reasonably small feature that adds some code does.

Reply:
Alright. BTW, are maintainers really always authors? For instance, there are debian maintainers of the dune package that are not authors.

Reply:
That's true for packages (since its a metadata overlay), but not (I think) ever true for source repos of the sort dune manages.  It does mean we'll need a package-specific maintainer field in the package manager areas.  So in the `(opam)` clause we could also allow a maintainer field in the future to override the global one in this PR.

I'm thinking that in the medium term dune or extensions can make smarter use of the global fields in order to improve error messages.  For instance a build failure could be automatically uploaded to a github issue in the right repo, or dune could even search online for messages from the `maintainers` who answer FAQs.

Reply:
Ok. That makes sense. So I guess the way to describe these fields is:
- maintainers: the people currently maintaining the project
- authors: everybody who has made significant contributions to the project